internal/infrastructure/kubernetes/ratelimit/resource.go

```diff
@@ -5,10 +5,117 @@
 
 package ratelimit
 
-import "fmt"
+import (
+	"fmt"
```

Comment:
why is this new file created ? another refactor ?

Reply:
Like proxy with the same structure and logic, the `resource` is the current module public resource logic.